### DISCUSSION TITLE
SONOFF Dual R3 - Power Monitoring - Set StartTotalTime issue

### DISCUSSION THREAD

**Discussion by MaartenTorfs** [arendst/Tasmota#18385] [Q&A]
According to the documentation, you can set the StartTotalTime time as epoch value. The optional 2nd parameter with EnergyToday/EnergyTotal and EnergyYesterday.

As I understand it correctly, you set the start time to Tuesday April 11, 2023 05:41:42 (Epoch = 1681191702) and reset the value of EnergyToday for channel 1 of the Dual R3 in the following way:

**EnergyToday1 0 1681191702**

The reset works but the second parameter does not seem to affect the TotalStartTime of the MQTT output as shown below. Is this normal? The documentation mentions "StartTotalTime" and the MQTT output "TotalStartTime", is that the same parameter?
What am I missing to set the start time?

thanks for helping me out

07:31:34.981 MQT: SONOFF/tele/DUALR3_PM06/SENSOR = {"Time":"2023-04-11T07:31:34","Switch1":"ON","Switch2":"ON","ENERGY":{"**TotalStartTime":"1970-01-14T15:24:55**","Total":[0.000,2.910],"Yesterday":[0.000,1.563],"Today":[0.000,1.347]

My Dual R3  runs on tasmota 12.4.0
used template = {"NAME":"Sonoff Dual R3 v2","GPIO":[32,0,0,0,0,0,0,0,0,576,225,0,0,0,0,0,0,0,0,0,0,3200,8128,224,0,0,0,0,160,161,0,0,0,0,0,0],"FLAG":0,"BASE":1}

**Comment by photocromax** [comment]
Have you tried to use epoch time in milliseconds? 
Just add 3 zeroes to your value

**Comment by arendst** [comment]
It's a confirmed (regression) bug!

I'll fix this ASAP

**Reply by arendst** [reply]
Fixed in latest dev branch
